examples/gltiles-hypsometric-relief.js

```diff
@@ -0,0 +1,141 @@
+import GlTiledTextureTerrainRGB from '../src/ol/source/GlTiledTexture/GlTiledTextureTerrainRGB.js';
+import Map from '../src/ol/Map.js';
+import TileLayer from '../src/ol/layer/Tile.js';
+import View from '../src/ol/View.js';
+import {GlTiles, XYZ} from '../src/ol/source.js';
+import {fromLonLat} from '../src/ol/proj.js';
+
+// This MapTiler API key is only valid for the OpenLayers examples page
+// Get your own!!
+const key = 'F0p8KPTwWvEEc5jKmckC';
```

Comment:
In other examples, we cloak API keys by using the `cloak` option in the template. See e.g. https://github.com/openlayers/openlayers/blob/bac6f4f6e892fbdc5a020ae23fc95f6539abc713/examples/bing-maps.html#L8-L10

Reply:
The good folks at @MapTiler (hey, @klokan !) let me know that they have an API key specific for openlayers examples. Now the GlTiles examples use the same key (and cloaking) than the `xyz-retina` example.